backend/routers/converters.py

```diff
@@ -52,11 +52,11 @@ async def image2png(file: UploadFile = File(...)):
     
     #failsafe checks
     if not file.content_type.startswith('image/'):
-        raise HTTPException(status_code=400, detail='File must be an image')
+        raise HTTPException(status_code=415, detail='File must be an image')
     if file.content_type == 'image/png':
-        raise HTTPException(status_code=400, detail='File is already a .png file')
+        raise HTTPException(status_code=403, detail='File is already a .png file')
```

Comment:
denne kan være bare 400, trenger ikke nødvendigvis bytte på der 400 er passende ;)

Reply:
This is fixed in  commit 1cd2b2e